invisible_cities/config/esmeralda.conf

```diff
@@ -21,4 +21,5 @@ paolina_params      = dict(
    strict_vox_size  = True,
    energy_threshold = 10 * keV,
    min_voxels       = 2,
-   blob_radius      = 21 * mm)
+   blob_radius      = 21 * mm,
+   max_num_hits     = 10000)
```

Comment:
Is this number optimized in some manner? You should make sure it doesn't kill real event while killing outliers

Reply:
We gave a look ad number of hits distribution, but in any case, this being a config file number, can easily be changed

Reply:
Yes, I mean for documentation kind of thing. It's good enough I guess